3.2/root/usr/share/container-scripts/mongodb/replset_supervisor.sh

```diff
@@ -53,25 +55,12 @@ if [[ "$1" == "initiate" ]]; then
   #       created on all members).
   sleep 10
 
-  # Some commands will force MongoDB client to re-connect. This is not working
-  # well in combination with '--eval'. In that case the 'mongo' command will fail
-  # with return code 254.
-  echo "=> Initiate Pod giving up the PRIMARY role ..."
-  mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --quiet --eval "rs.stepDown(120);" &>/dev/null || true
-
-  # Wait till the new PRIMARY member is elected
-  echo "=> Waiting for the new PRIMARY to be elected ..."
-  mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --quiet --host ${mongo_node} --eval "var done=false;while(done==false){var members=rs.status().members;for(i=0;i<members.length;i++){if(members[i].stateStr=='PRIMARY' && members[i].name!='$(mongo_addr)'){done=true}};sleep(500)};" &>/dev/null
-
-  # Remove the initialization container MongoDB from cluster and shutdown
-  echo "=> A new PRIMARY member was elected, shutting down local mongod ..."
-  mongo_remove
-
-  mongod -f ${MONGODB_CONFIG_PATH} --shutdown &>/dev/null
-  wait_for_mongo_down
 
   echo "=> Successfully initialized replSet"
 
+  # Exit this pod
+  kill ${main_process_id}
```

Comment:
Hmm, sorry I didn't follow. Why are we going to kill `run-mongod-replication`? How does this relate to https://bugzilla.redhat.com/show_bug.cgi?id=1260586?

(BTW, if this PR intends to fix that bug, please add a link in the PR's description)